src/SpeedDial/speeddial.st.css

```diff
@@ -0,0 +1,63 @@
+@st-import [--spacing-unit] from "../styles/default/project.st.css";
```

Comment:
It would be nice to use camelCase for the filename. Instead of `speeddial.st.css` it could be `speedDial.st.css`.